src/agent/src/image_rpc.rs

```diff
@@ -136,9 +189,15 @@ impl ImageService {
             verify_cid(cid)?;
         }
 
+        if aa_kbc_params != "" && aa_started {
```

Comment:
`! aa_started` ?  the same as above
and `aa_kbc_params.is_empty()`. Otherwise static-check will fail.